ml-agents/mlagents/trainers/torch/utils.py

```diff
@@ -284,3 +284,13 @@ def get_probs_and_entropy(
         else:
             all_probs = torch.cat(all_probs_list, dim=-1)
         return log_probs, entropies, all_probs
+
+    @staticmethod
+    def masked_mean(tensor: torch.Tensor, masks: torch.Tensor) -> torch.Tensor:
```

Comment:
A tiny unit test for it? Making sure that if all the masks are 0 it does not 💥

Reply:
It will totally 💥, good call

Reply:
Added test and fixed 💥 issue